assets/component-mega-menu.css

```diff
@@ -9,11 +9,10 @@
   border-right: 0;
   left: 0;
   overflow-y: auto;
-  padding-bottom: 2.4rem;
-  padding-top: 2.4rem;
+  padding-bottom: 3rem;
+  z-index: 6;
   position: absolute;
   right: 0;
-  top: 100%;
```

Comment:
I added back top: 80% to get it a tad bit closer to the L1 link to give the illusion that it is opening under the link. I found that top: 100% gave slightly more space between the mega menu and L1 links. I tested this change with changing the window size and didn't find an overlap. Let me know with the new code if you find any overlaps :)

Reply:
@danielvan i guess we would have to add a separator line at the top of the mega menu.
Also how do you feel about the mega menu hiding content inside the Header? I feel like previous version was a lot safer but maybe it's just me!

Reply:
I agree with the line. The con of the previous version is that it would always open under the nav right?

I'm ok in hiding content because the user interacted with it.
It feels better to me than the alternative where it can open completely disconnected with where the user interacted.

Reply:
For it to cut other elements seem odd to me. If it wasn't full width why not but if it's full width it shouldn't imo. 
It isn't more connected to the nav that way I find and cutting the logo a little bit too: 

<details>
<summary>Screenshot</summary>

![](https://screenshot.click/18-35-0tyyj-k6mmr.png)
</details>